{"name": "test_get_users_order_list_with_auth_success", "status": "passed", "description": "Проверка невозможности получения списка заказов, если пользователь не авторизован.Получаем код 401", "steps": [{"name": "Получение списка заказов конкретного пользователя", "status": "passed", "parameters": [{"name": "token_user", "value": "''"}], "start": 1717267068104, "stop": 1717267068317}], "start": 1717267068103, "stop": 1717267068318, "uuid": "969bdc01-0001-4813-a4a1-3bcea7086ebc", "historyId": "68d62bbe8b9278a5a1a1c7f4457858dd", "testCaseId": "68d62bbe8b9278a5a1a1c7f4457858dd", "fullName": "tests.test_get_users_order_list.TestGetUsersOrderList#test_get_users_order_list_with_auth_success", "labels": [{"name": "story", "value": "Ручка /api/orders"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_users_order_list"}, {"name": "subSuite", "value": "TestGetUsersOrderList"}, {"name": "host", "value": "WIN-BHU1OVNKOHE"}, {"name": "thread", "value": "13424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_users_order_list"}]}